sdk/identity/Azure.Identity/src/IdentityClientOptions.cs

```diff
@@ -12,19 +12,11 @@ public class IdentityClientOptions : ClientOptions
         private readonly static Uri DefaultAuthorityHost = new Uri("https://login.microsoftonline.com/");
         private readonly static TimeSpan DefaultRefreshBuffer = TimeSpan.FromMinutes(2);
 
-        public RetryPolicy RetryPolicy { get; set; }
-
         public Uri AuthorityHost { get; set; }
 
         public IdentityClientOptions()
         {
             AuthorityHost = DefaultAuthorityHost;
-            RetryPolicy = new RetryPolicy()
-            {
-                Mode = RetryMode.Exponential,
-                Delay = TimeSpan.FromMilliseconds(800),
-                MaxRetries = 3
-            };
```

Comment:
Is there still a way for the service client to specify it's own defaults for retry?  or this is all shared?

Reply:
Yes, by setting a property in client options constructor.

``` C#
public ConfigurationClientOptions()
{
    RetryPolicy.MaxRetries = 10;
}
```

Reply:
Is there a plan to document different Azure.Core scenarios for people to know? like this one?

Reply:
For client SDK authors? Not that I know of.

Reply:
Yes. It could be useful. also having like samples with best practices or something.

Could someone outside our team create a client sdk using Azure.Core? How can we guide them?